WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -579,7 +579,7 @@ private void SetFeesAndTexts()
 				var d = feeTarget / Constants.OneDayConfirmationTarget;
 				ConfirmationExpectedText = $"{d} {IfPlural(d, "day", "days")}";
 			}
-			else if (feeTarget == 10008)
+			else if (feeTarget == 1008)
```

Comment:
I think this is meant to be set to `Constants.SevenDaysConfirmationTarget`, good catch

Reply:
Given the displayed text is "two weeks", it should be:
```suggestion
			else if (feeTarget == 2 * Constants.SevenDaysConfirmationTarget)
```

If this is correct then there will be a gap and the previous condition should be also updated as

```c#
else if (feeTarget >= 145 && feeTarget < 2 * Constants.SevenDaysConfirmationTarget) // days
```

Good catch!

Reply:
I used the const, and yes I think maybe we should say `one week` not `two weeks™`.
Notice also that there is `™` after two weeks, I don't know why.

Reply:
the `tm` is there a joke/meme

Reply:
Hehe, @lontivero, are you again taking the fun out of the code?! :smile: